js/modifiers.js

```diff
@@ -191,6 +195,7 @@ var modifiersManagement = {
                 .filter(element => !this.modifiersDrawPile.includes(element))
 
             if(availableCurses.length > 0) this.addModifier(availableCurses[0])
+            this.shuffle();
         },
         resetModifiers: function() {
             this.shuffleModifiersDeck()
```

Comment:
Do you remember what the commented line below is about? Should we remove it or was it something important?

Reply:
Github doesnt show it in the preview, but line 202 is : 
`//this.modifiersDrawPile.forEach(card => {if(this.checkIfCurseOrBless(card)) this.removeModifier(card)})`